MetaCell/nwb-explorer
# Feature/jupyter backend

# What's new
Uses Jupyter as backend, adding the notebook capability.

# How to test

- start a new conda env (suggested)
- run utilities/install.py 
- run script ./NWBE
- go to localhost:8888/geppetto?nwbfile=https://github.com/OpenSourceBrain/NWBShowcase/raw/master/NWB/time_series_data.nwb

See also 
- https://github.com/tarelli/geppetto-nwbexplorer/pull/2
- https://github.com/openworm/org.geppetto.frontend.jupyter/pull/62
